keps/sig-node/compute-device-assignment.md

```diff
@@ -63,16 +63,30 @@ In this document we will discuss the motivation and code changes required for in
 As soon as this interface has been introduced it was used by CNI plugins like [kuryr-kubernetes](https://review.opendev.org/#/c/651580/) in couple with [intel-sriov-device-plugin](https://github.com/intel/sriov-network-device-plugin) to correctly define which devices were assigned to the pod.
 
 ### Topology aware scheduling
-This interface can be used to collect allocated resources with information about the NUMA topology of the worker node. This information can then be used in NUMA aware scheduling.
+This interface can be used to collect allocated resources with information about the NUMA topology of the worker node.
+This interface can be used to the available resources on the worker node. The kubelet is the best source of information because it manages the device plugins.
+The information can then be used in NUMA aware scheduling.
 
 ## Changes
 
-Add a v1alpha1 Kubelet GRPC service, at /var/lib/kubelet/pod-resources/kubelet.sock, which returns information about the kubelet's assignment of devices and cpus with NUMA id. It obtains this information from the internal state of the kubelet's Device Manager and CPU Manager respectively. The GRPC Service returns a single PodResourcesResponse, which is shown in proto below:
+Add a v1alpha1 Kubelet GRPC service, at `/var/lib/kubelet/pod-resources/kubelet.sock`, which returns information about
+- the devices which kubelet knows about from the device plugins.
+- the kubelet's assignment of devices and cpus with NUMA id to containers.
+The GRPC service obtains this information from the internal state of the kubelet's Device Manager and CPU Manager respectively.
+The GRPC Service is described in proto below:
 ```protobuf
 // PodResources is a service provided by the kubelet that provides information about the
 // node resources consumed by pods and containers on the node
 service PodResources {
     rpc List(ListPodResourcesRequest) returns (ListPodResourcesResponse) {}
+    rpc ListDevices(ListDevicesRequest) returns (ListDevicesResponse) {}
```

Comment:
should be `ListResources`, fixing

Reply:
Thinking out loud here, given memory manager is going to be introduced, hugepage information i.e. capacity configured on a node and allocated hugepages (along with its topology info) might be something we would like to obtain via podresource API too. ListResources might be more suitable for cases where resources (in addition to cpu and devices) will be tracked by pod resource API, alternatively this could mean that another service eg ListHugepages but this might not be needed if we assume hugepage capacity won't change dynamically once configured on the node.

Reply:
I agree. I'm going to use "Resources" to be more generic, and I believe also a bit more consistent with the rest of the API here.